src/main/java/racingcar/v2/domain/car/Position.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.v2.domain.car;
+
+import java.util.Objects;
+
+public class Position implements Comparable<Position>{
+    private int position;
+
+    public Position() {
+        this(0);
+    }
+
+    public Position(String position) {
+        this(Integer.parseInt(position));
+    }
+
+    public Position(int position) {
+        if (position < 0) {
+            throw new IllegalArgumentException("이동거리는 음수가 될수 없습니다.");
+        }
```

Comment:
매직넘버와 매직리터럴은 분리해주세요. 
```suggestion
        if (position < MIN_LIMIT_POSITION) {
            throw new IllegalArgumentException(INVALID_POSITITION_MESSAGE);
        }
```